nixos/tests/freeswitch.nix

```diff
@@ -0,0 +1,36 @@
+import ./make-test-python.nix ({ pkgs, ...} : {
+  name = "freeswitch";
+  meta = with pkgs.stdenv.lib.maintainers; {
+    maintainers = [ misuzu ];
+  };
+
+  nodes = {
+    freeswitch = { config, lib, ... }: {
+      networking.useDHCP = false;
+      networking.interfaces.eth1 = {
+        ipv4.addresses = [
+          {
+            address = "192.168.0.1";
+            prefixLength = 24;
+          }
+        ];
+      };
+      services.freeswitch = {
+        enable = true;
+        configDir = {
+          # mod_sofia is crashing FreeSWITCH with STUN when there is no internet connection
+          # https://github.com/signalwire/freeswitch/issues/201#issuecomment-570196382
+          "vars.xml" = (
+            builtins.replaceStrings ["stun:stun.freeswitch.org"] ["192.168.0.1"] (
+              builtins.readFile "${config.services.freeswitch.package}/etc/freeswitch/vars.xml"));
```

Comment:
This is import-from-derivation, meaning that the evaluation depends on a build. This isn't allowed in nixpkgs as it messes up hydra.

Reply:
I removed ```builtins.readFile``` part. Is current approach allowed?